tests/mtest/src/main/scala/tests/PCSuite.scala

```diff
@@ -100,7 +100,7 @@ trait PCSuite {
       case NonFatal(e) =>
         println(s"warn: ${e.getMessage}")
     }
-    workspace.inputs(filename) = (code2, dialect)
+    workspace.inputs(file.toURI.toString()) = (code2, dialect)
```

Comment:
Pure curiosity, why did that change?

Reply:
That was actually showing errors whenever running tests, wasn't really problematic, but spamming the logs.